docs/user-guide/media-editor-window.md

```diff
@@ -0,0 +1,93 @@
+# Media Editor Window
+**********************
+
+The **Change media source** button allows to access the Media Editor Window:
+
+<img src="../img/media-editor-window/media-editor-window.jpg" class="ms-docimage"/>
+
+Through this window, you can add or edit three different types of media:
+
+* Images
+
+* Videos 
+
+* Maps
+
+## Images
+
+In order to add an image, the user can click on the **Images** button <img src="../img/button/images.jpg" class="ms-docbutton"/> for switch to the images section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the image settings.
+
+<img src="../img/media-editor-window/img-settings.jpg" class="ms-docimage"/>
+
+In particular, it is possible to insert the following parameters:
+
+* The **Source** of the image (link to find that resource)
+
+* The **Title** of the image
+
+* The **Alternative text**, that appears if the link is broken
+
+* A **Description**, available from the image settings and in preview
```

Comment:
@ElenaGallo please reformilate it better and explain how the description is used and where in the story by including also a screenshot